indico/htdocs/js/indico/modules/sessions.js

```diff
@@ -0,0 +1,134 @@
+(function(global) {
+    'use strict';
+
+    function handleRowSelection() {
+        $('table.i-table input.select-row').on('change', function() {
+            $(this).closest('tr').toggleClass('selected', this.checked);
+            $('.js-requires-selected-row').toggleClass('disabled', !$('.sessions input:checkbox:checked').length)
+        }).trigger('change');
+    }
+
+    function setupTableSorter() {
+        $('.sessions .tablesorter').tablesorter({
+            cssAsc: 'header-sort-asc',
+            cssDesc: 'header-sort-desc',
+            cssInfoBlock: 'avoid-sort',
+            cssChildRow: 'session-blocks-row',
+            headerTemplate: '',
+            sortList: [[1, 0]]
+        });
+    }
+
+    function setupPersonsAjaxDialog() {
+        $('.session-persons-list').ajaxDialog();
+    }
+
+    global.setupSessionsList = function setupSessionsList() {
+        handleRowSelection();
+        setupTableSorter();
+        setupPersonsAjaxDialog();
+
+        $('#select-all').on('click', function() {
+            $('table.i-table input.select-row').prop('checked', true).trigger('change');
+        });
+
+        $('#select-none').on('click', function() {
+            $('table.i-table input.select-row').prop('checked', false).trigger('change');
+        });
+
+        $('.sessions .toolbar').on('click', '.disabled', function(evt) {
+            evt.preventDefault();
+            evt.stopPropagation();
+        });
+
+        $('#add-new-session').on('click', function() {
+            var $this = $(this);
+            ajaxDialog({
+                url: $this.data('href'),
+                title: $this.data('title'),
+                onClose: function(data) {
+                    if (data) {
+                        $('.sessions-wrapper').html(data.sessions_list);
+                        handleRowSelection();
```

Comment:
But we don't need it in all cases. Do you want it to conform to the rest of the code?